### ISSUE TITLE
Obsolete `--namedExport` option

### ISSUE THREAD

**Issue by mizdra** [mizdra/enhanced-typed-css-modules#109]
- `--namedExport` option restricts the class name
  - In `css-loader`, `localsConvention` only allows `camelCaseOnly` and `dashesOnly`.
  - ref: https://github.com/webpack-contrib/css-loader#namedexport
- This restriction is also true for `enhanced-typed-css-modules`.
- `--namedExport` option also restricts definition jumps.
  - ref: https://github.com/mizdra/checkable-css-modules/blob/cc9ebcc1533d60a44aaa2fc0c1f23b1a6264bb9a/src/emitter/dts.ts#L66-L67
- In summary, `--namedExport` option does not take full advantage of the full functionality of css modules and confuses the user.
- Therefore, we will obsolete `--namedExport` option.

## Memo
- `--namedExport` option is not implemented in [`postcss-modules`](https://github.com/madyankin/postcss-modules) for now.
